Classes/Event/EventPublisher.php

```diff
@@ -104,9 +104,7 @@ public function publishMany(string $streamName, array $events, int $expectedVers
         $eventStore = $this->eventStoreManager->getEventStoreForStreamName($streamName);
         $rawEvents = $eventStore->commit($streamName, $convertedEvents, $expectedVersion);
 
-        $configuration = new PropertyMappingConfiguration();
-        $configuration->allowAllProperties();
-        $configuration->forProperty('*')->allowAllProperties();
+        $configuration = new AllowAllPropertiesPropertyMappingConfiguration();
```

Comment:
Ah, I like that!
So much that I would even imagine this to be part of Flow!?

So much even, that I could imagine some special syntax like
```php
$this->propertyMapper->forceConvert($source, $targetType);
```

(but that could be a separate step ofc)

Reply:
Yes! Let's have a separate PR for that.